src/main/kotlin/org/elm/workspace/commandLineTools/ElmCLI.kt

```diff
@@ -17,13 +17,11 @@ import java.nio.file.Path
  */
 class ElmCLI(private val elmExecutablePath: Path) {
 
-    // TODO [kl] allow the caller to specify the main entry point Elm file
-
-    fun make(owner: Disposable, elmProject: ElmProject): ProcessOutput {
+    fun make(owner: Disposable, elmProject: ElmProject, path: String): ProcessOutput {
```

Comment:
please use Java's `Path` type instead of a string for consistency with the rest of the plugin.